websurvey/lib/typewheels/responseStore.js

```diff
@@ -12,6 +15,15 @@ class ResponseStore {
   getQa() {
     return this.qa;
   }
+
+  validateFieldValue(field, fieldValue, isRequired) {
+    const res = v.validator(field)(fieldValue);
+    if (!res.valid || (isRequired && fieldValue === " ")) {
```

Comment:
Worth maybe checking for other types of empty strings! Anything that's all whitespace characters, for example.

Reply:
Also, why not put this logic into the "validator" itself? Seems like a reasonable place to have, rather than have some logic for a valid answer inside that function and some outside!